python/grass/app/cli.py

```diff
@@ -66,6 +70,25 @@ def subcommand_run_tool(args, tool_args: list, print_help: bool):
                 return error.returncode
 
 
+def subcommand_create_project(args) -> int:
+    """Translates args to function parameters"""
+    try:
+        gs.create_project(
+            path=args.path,
+            crs=args.crs,
+            proj4=args.proj4,
+            wkt=args.wkt,
+            datum=args.datum,
+            datum_trans=args.datum_trans,
+            desc=args.description,
```

Comment:
Change after merging #6440 
```suggestion
            description=args.description,
```